pandas/tests/io/excel/test_readers.py

```diff
@@ -736,39 +735,44 @@ class TestExcelFileRead:
         pytest.param(None, marks=pytest.mark.skipif(
             not td.safe_import("xlrd"), reason="no xlrd")),
     ])
-    def cd_and_set_engine(self, request, datapath, monkeypatch):
+    def cd_and_set_engine(self, request, datapath, monkeypatch, read_ext):
         """
         Change directory and set engine for ExcelFile objects.
         """
+        if request.param == 'openpyxl' and read_ext == '.xls':
+            pytest.skip()
+
         func = partial(pd.ExcelFile, engine=request.param)
         monkeypatch.chdir(datapath("io", "data"))
         monkeypatch.setattr(pd, 'ExcelFile', func)
 
     def test_excel_passes_na(self, read_ext):
 
-        excel = ExcelFile('test4' + read_ext)
+        excel = pd.ExcelFile('test4' + read_ext)
```

Comment:
We were monkeypatching `pd.ExcelFile` but used `ExcelFile` imported directly in tests which was causing this to run exclusive of the specified engine. Figured easiest just to stick with the pd namespace